content/en/code_analysis/software_composition_analysis/_index.md

```diff
@@ -22,16 +22,16 @@ algolia:
   tags: ['software composition analysis', 'datadog software composition analysis', 'library vulnerabilities', 'SCA']
 ---
 
+{{< callout url="#" btn_hidden="true" header="Join the Preview!" >}}
+Software Composition Analysis is in Preview.
```

Comment:
We are changing it from `Code Analysis` to `Software Composition Analysis`. Is it fine?

Reply:
Yes, that change isn't happening until 2025.